sql/driver/conn.go

```diff
@@ -139,3 +140,14 @@ func (c *conn) send(args Request) (*rows, error) {
 	}
 	return r, nil
 }
+
+// Execute all the URL settings against the db to create
+// the correct session state.
+func (c *conn) applySettings(q url.Values) error {
+	if db := q["database"]; len(db) > 0 {
+		if _, err := c.Exec("SET DATABASE = "+db[0], []driver.Value{}); err != nil {
```

Comment:
do you need the empty slice? i think you can just omit it. also, how come db[0]?